src/plugin.css

```diff
@@ -1 +1 @@
p:1px;width:auto;height:auto}.config-container .single-page{margin-top:2em}.config-container .config-highlight{color:#888;cursor:pointer;font-weight:200}.config-container .config-highlight .mch-name{color:#666;font-weight:300}.config-label{font-size:1.1em;font-weight:600;margin-bottom:.8em;text-transform:uppercase}.config-intro{font-size:1em;font-weight:200;line-height:1.5;padding-left:.35em;width:70%}.config-list{margin:2em 0 0 0}.config-list li{border:1px solid #dedede;overflow:hidden}.config-list .config-name{color:#666;cursor:pointer;font-size:1.5em;font-weight:300}.config-list.list li{margin-bottom:2em;position:relative;padding:1em}.config-list.list .config-name{display:inline-block;width:30em}.config-list.list .config-status{display:inline-block;position:absolute;right:5em;top:3px}.config-list.list .config-highlights{display:inline-block;margin-left:7em}.config-list.list .config-highlight{display:inline-block;margin-left:1em;width:11em}.config-list.grid li{display:inline-block;height:14em;margin:0 2.5em 2.5em 0;padding:0;position:relative;text-align:center;width:15em}.config-list.grid li:hover{box-shadow:0 0 3px -1px #000}.config-list.grid .config-name{line-height:1.5;padding:12px 0 0 0;height:145px}.config-list.grid .config-name.with-highlight{height:90px}.config-list.grid .config-status{border-top:1px solid #dedede;position:absolute;bottom:0;width:100%}.config-list.grid .config-highlight{margin-bottom:10px}.config-name label{font-weight:300;margin-bottom:1em}.config-empty{font-size:1.3em;font-weight:400;padding:5em 2em;text-align:center}.create-config-button{color:#eee;float:right;font-size:1.3em;padding:.4em .6em}.config-enable-message p{font-size:1.2em;text-align:center}.config-input{display:inline-block}.toggle-label{display:block;cursor:pointer;background:#f3f3f3;height:15px;width:35px;border-radius:15px;margin:15px auto;position:relative;box-shadow:0 0 0 2px #dddddd;font-weight:300;transition:.25s ease-in-out}.toggle-label .on{display:block;position:absolute;z-index:0;left:0;line-height:15px;opacity:0;color:#13bf11;transition:.25s ease-in-out}.toggle-label .off{display:block;position:absolute;z-index:0;right:45px;text-align:right;line-height:15px;opacity:1;color:#f90;transition:.25s ease-in-out}.toggle-label:before{content:'';display:block;position:absolute;z-index:1;top:0;left:0;border-radius:15px;height:15px;width:15px;background:white;box-shadow:0 3px 3px rgba(0,0,0,0.2),0 0 0 2px #ddd;transition:.25s ease-in-out}.toggle{position:absolute;left:-5000px}.toggle+.toggle-label{background:#f90;box-shadow:0 0 0 2px #f90}.toggle+.toggle-label:before{box-shadow:0 3px 3px rgba(0,0,0,0.2),0 0 0 2px #f90;transition:.25s ease-in-out}.toggle:checked+.toggle-label{background:#13bf11;box-shadow:0 0 0 2px #13bf11}.toggle:checked+.toggle-label .on{left:45px;opacity:1}.toggle:checked+.toggle-label .off{right:0;opacity:0}.toggle:checked+.toggle-label:before{left:19px;box-shadow:0 3px 3px rgba(0,0,0,0.2),0 0 0 2px #13bf11;transition:.25s ease-in-out}
```

Comment:
@alexweber so we need to commit compiled css because mayan doesn't do it for node modules? Similar reason mayan doesnt transpile the js?